script.js

```diff
@@ -0,0 +1,81 @@
+const inputField = document.querySelector(".input-section");
+const checkInputs = document.querySelectorAll(".check-section-inputs");
+const passwordFieldOne = document.querySelector("#input-password-one");
+const passwordFieldTwo = document.querySelector("#input-password-two");
+const btnShowHide = document.querySelector("#show-hide-btn");
+const lowerCase = document.querySelector("#lowercase-check");
+const toUpperCase = document.querySelector("#uppercase-check");
+const includesNumber = document.querySelector("#number-check");
+const atLeast10Char = document.querySelector("#at-least-check");
```

Comment:
Gute Variablenbenennung

Gerne überlegen, welche Variablen in welchem Scope benötigt werden